src/memory_pool.c

```diff
@@ -59,6 +59,7 @@ static umf_result_t umfPoolCreateInternal(const umf_memory_pool_ops_t *ops,
     }
 
     *hPool = pool;
+    LOG_INFO("Memory pool created: %p", (void *)pool);
```

Comment:
I would like to keep one convention of prefixes.... sometimes you use a function name and sometimes you don't. Perhaps you could use "__FUNCTION__"?

Reply:
Done. I changed `LOG_*` macros and added `__func__` to it to automatically add function name to the log message.